src/main/java/org/ow2/proactive/catalog/report/CatalogObjectReportPDFGenerator.java

```diff
@@ -80,62 +100,150 @@
+        }
     }
 
-    private void populateTable(PDPage page, BaseTable dataTable,
-            SortedSet<CatalogObjectMetadata> orderedObjectsPerBucket) throws IOException {
-        //Create the data
-        List<List> data = new ArrayList();
-        data.add(new ArrayList<>(HEADER));
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData) {
+        Cell<PDPage> bucketNameCell = row.createCell(width, catalogObjectData);
+        bucketNameCell.setFontSize(6);
+        bucketNameCell.setAlign(HorizontalAlignment.CENTER);
+    }
+
+    private void createSecondaryHeader(BaseTable table) {
+        Row<PDPage> row = table.createRow(15f);
+        row.createCell(75, "Source:");
+        row.createCell(25, schedulerUrl + "/catalog");
+    }
+
+    private void createMainHeader(BaseTable table) throws IOException {
+        Row<PDPage> headerRow = table.createRow(15f);
 
-        addRowForEachObject(orderedObjectsPerBucket, data);
+        URL url = new URL(schedulerUrl + ACTIVEEON_LOGO);
+        BufferedImage imageFile = ImageIO.read(url);
+        headerRow.createImageCell((100 / 8f), new Image(imageFile));
 
-        DataTable t = new DataTable(dataTable, page);
-        styleHeader(t);
-        t.addListToTable(data, DataTable.HASHEADER);
-        dataTable.draw();
+        Cell<PDPage> cell = headerRow.createCell((100 / 8f) * 7, MAIN_TITLE);
+        cell.setFillColor(java.awt.Color.decode("#EE7939"));
+        cell.setTextColor(java.awt.Color.decode("#0E2C65"));
 
+        table.addHeaderRow(headerRow);
     }
 
-    private void addRowForEachObject(SortedSet<CatalogObjectMetadata> orderedObjectsPerBucket, List<List> data) {
+    private void createDataHeader(BaseTable table) {
+        Row<PDPage> factHeaderrow = table.createRow(15f);
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Bucket Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Project Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Object Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f) * 2, "Description");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Kind");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Content Type");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Icon");
+        table.addHeaderRow(factHeaderrow);
+    }
 
-        String currentBucketName = "";
+    private void createDataHeaderCell(Row<PDPage> factHeaderrow, float width, String title) {
+        Cell<PDPage> cell;
+        cell = factHeaderrow.createCell(width, title);
+        cell.setFontSize(6);
+        cell.setFillColor(java.awt.Color.decode("#F3F3F4"));
+        cell.setAlign(HorizontalAlignment.CENTER);
+    }
 
+    private List<String[]> getCatalogObjects(SortedSet<CatalogObjectMetadata> orderedObjectsPerBucket)
+            throws IOException {
+        //Create the data
+        List<String[]> data = new ArrayList<String[]>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)